schema.md

```diff
@@ -1,82 +1,71 @@
-# Design for netcdf storage format for mcmc traces
+# InferenceData schema specification
+The `InferenceData` schema scheme defines a data structure compatible with [NetCDF](https://www.unidata.ucar.edu/software/netcdf/) with 3 goals in mind, usefulness in the analysis of Bayesian inference results, reproducibility of Bayesian inference analysis and interoperability between different inference backends and programming languages.
 
-/
-All data relating to a mcmc run.
+Currently there are 2 implementations of this design:
+* [ArviZ](https://arviz-devs.github.io/arviz/) in Python which integrates with:
+  - [emcee](https://emcee.readthedocs.io/en/stable/)
+  - [PyMC3](https://docs.pymc.io)
+  - [pyro](https://pyro.ai/)
+      and [numpyro](https://pyro.ai/numpyro/)
+  - [PyStan](https://pystan.readthedocs.io/en/latest/index.html),
+      [CmdStan](https://mc-stan.org/users/interfaces/cmdstan)
+      and [CmdStanPy](https://cmdstanpy.readthedocs.io/en/latest/index.html)
+  - [tensorflow-probability](https://www.tensorflow.org/probability)
+* [ArviZ.jl](https://github.com/sethaxen/ArviZ.jl) in Julia which integrates with:
+  - [Turing](https://turing.ml/dev/).
 
-attrs:
-backend: "stan" or "pymc3"
-version: str
-model_name?: str
-comment?: str
-model_version?: str
-timestamp: int
-author?: str
+## Current design
+`InferenceData` stores all quantities relevant in order to fulfill its goals in different groups. Each group, described below, stores a conceptually different quantity generally represented by several multidimensional labeled variables.
 
-/coords
-For each dimension, we store a corresponding variable here, that contains the labels for that dimension.
+Each group should have one entry per variable, with the first two dimensions of each variable should be the sample identifier (`chain`, `draw`). Dimensions must be named and explicit their index values, called coordinates. Coordinates can have repeated identifiers and may not be numerical. Variable names must not share names with dimensions.
```

Comment:
I'm being picky, but for some of these groups (e.g., `observed_data` and `constant_data`), chain and draw don't make sense.

Also, in the following sentence, I think that you mean "specify" instead of "explicit."

One last point: how can coordinates have repeated identifiers?  Wouldn't that break indexing by coordinate?

Reply:
I will correct the wording, thanks!

Regarding repeated identifiers, the radon example has repeated identifiers. This has some issues when using `sel` and `where` should be used instead generally, but for most plots it is not an issue

Reply:
Thanks, I didn't know that.